Mobile/ShopLens/ImageRecognitionMobile/Classificators/WebClassificator.cs

```diff
@@ -17,6 +17,17 @@ namespace ImageRecognition.Classificators
     /// </summary>
     public class WebClassificator : IImageClassificator, IAsyncImageClassificator
     {
+        private readonly Guid projectId;
+        private readonly string predictionKey;
+        private readonly string uri; 
+
+        public WebClassificator(string passProjectId, string passPredictionKey, string passRequestUri)
+        {
+            projectId = Guid.Parse(passProjectId);
+            predictionKey = passPredictionKey;
+            uri = passRequestUri;
```

Comment:
I think these should be loaded by the WebClassificator class instead of being passed through the constructor? Or maybe they should be passed to the ClassifyImage method. 

What does the "pass-" prefix in the names mean? Like in _pass_ProjectId?

Reply:
It means that the variable was passed to this method.

Reply:
Okay, I can make it so that they are passed to the ClassifyImage method.

Reply:
But why does it make a difference, I don't really understand.

Reply:
Huh, right now we're calling ClassifyImageAsync from the CameraActivity class and not ClassifyImage. So, now I would need to add those variables to both method definitions? Seems rather unnecessary.

Reply:
And now also the interfaces must be changed.

Reply:
We don't write "pass" before every parameter name in a function, not sure why we should here.

I think that the WebClassificator class should handle everything, just like TensorFlow classificator handled it's own model. The user of the WebClassificator doesn't care what projectId or predictionKey or Uri is. The user will just have to pass the same thing every time

Reply:
Yeah, I just wrote pass so that I wouldn't have to write this.x = x. 
Should you come up with different names for the passed parameters or write this in such situations in C#? This might seem like a dumb question, but I never saw this.x = x written anywhere since we started this project.

Reply:
In other words, should you avoid "qualifying members hidden by similar names" with the this keyword in C#?

Reply:
Is this one of those religious questions of C#?